### PR TITLE
Pass `trailingSlash` from Next.js config to firebase.json

### PR DESCRIPTION
**Pull request by leoortizz**

### CODE REVIEW

File: src/frameworks/next/index.ts [hunk 1]
```diff
@@ -95,7 +95,11 @@ export async function build(dir: string): Promise<BuildResult> {
   });
 
   const reasonsForBackend = [];
-  const { distDir } = await getConfig(dir);
+  const {
+    distDir,
+    // trailingSlash defaults to false in Next.js: https://nextjs.org/docs/api-reference/next.config.js/trailing-slash
+    trailingSlash = false,
```
Comment (jamesdaniels) [inline src/frameworks/next/index.ts:101]: can we push the default into getConfig?

Comment (leoortizz) [reply src/frameworks/next/index.ts:101]: I tried using the third parameter in `loadConfig` but ends up that it [overrides the user config](https://github.com/vercel/next.js/blob/440560fbbb26d06233b41738c1e99e246291e7e7/packages/next/src/server/config.ts#L878). I'll dig into it a little bit more

Comment (jamesdaniels) [reply src/frameworks/next/index.ts:101]: I'm thinking here https://github.com/firebase/firebase-tools/blob/e02f4fd1f13d68df66551ff6e014327c34b18e5a/src/frameworks/next/index.ts#L451

Comment (leoortizz) [reply src/frameworks/next/index.ts]: gotcha, the same way as `distDir`.. I missed it. Just pushed it

**Review by jamesdaniels — APPROVED** [review]
LGTM, add changelog

**Pull request merged**